components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticatorConstants.java

```diff
@@ -59,6 +59,7 @@ public abstract class BasicAuthenticatorConstants {
     public static final String SHOW_PENDING_USER_INFORMATION_CONFIG = "ShowPendingUserInformation";
     public static final String IS_INVALID_USERNAME = "isInvalidUsername";
     public static final String USERNAME_USER_INPUT = "usernameUserInput";
+    public static final String LOGIN_HINT = "login_hint";
```

Comment:
Cannot we directly use `INPUT_TYPE_LOGIN_HINT` constant in https://github.com/wso2/carbon-identity-framework/blob/0b619165b9c47ff76f7cff2db45e3c31da17a00e/components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java#L158 without introducing a separate constant

Reply:
Do we need this variable?